src/Razor/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X/src/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X.csproj

```diff
@@ -9,6 +9,8 @@
     <IsShipping>false</IsShipping>
     <!-- Need to build this project in source build -->
     <ExcludeFromSourceBuild>false</ExcludeFromSourceBuild>
+    <!-- No need to track public APIs of this design-time build infrastructure. -->
+    <AddPublicApiAnalyzers>false</AddPublicApiAnalyzers>
```

Comment:
@JunTaoLuo @pranavkm do you agree with this assessment❔ Given the few references to this project and Microsoft.AspNetCore.Mvc.Razor.Extensions.Version2_X.csproj, it's not clear to me how they're used.

Reply:
Yup. This isn't required. The assembly is only referenced by the SDK / razor compiler and does not get shipped.